packages/pigeon/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 16.0.1
+
+* Separates message call code generation into separate methods in the `KotlinGenerator`.
```

Comment:
This should start with [kotlin] instead of ending with "in the `kotlinGenerator`"